### ISSUE TITLE
add a much longer timeout value to waiting for the console on `gofabric8 start`

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/gofabric8#311]
to avoid `Could not find finalized endpoint being pointed to by fabric8` messages making folks think there's an issue when really its just taking a while to download all the docker images

**Comment by jimmidyson** [comment]
Perhaps poll or watch until pods are ready?

**Comment by jstrachan** [comment]
yeah - I'd be happy just removing the timeout completely really & just keep polling for it to be up

**Comment by jstrachan** [comment]
as a workaround folks can watch docker images download via:
```
kubectl get pod -w
```
